testing/MLDB-1645_list_files_test.py

```diff
@@ -0,0 +1,33 @@
+#
+# MLDB-1645_list_files_test.py
+# Mich, 2016-05-11
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+#
+
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+class ListFilesTest(MldbUnitTest):  # noqa
+
+    def test_it(self):
+        mldb.post('/v1/procedures', {
+            'type' : 'list.files',
+            'params' : {
+                'path' : 'file://./mldb/testing/',
+                'outputDataset' : {
+                    'id' : 'output',
+                    'type' : 'tabular'
+                },
+                'runOnCreation' : True
+            }
+        })
+
+        res = mldb.query("""
+            SELECT * FROM output WHERE
+            rowName()='file://./mldb/testing/MLDB-1645_list_files_test.py'
+        """)
+        self.assertEqual(len(res), 2)
+        self.assertEqual(
+            res[1][0], '"file://./mldb/testing/MLDB-1645_list_files_test.py"')
```

Comment:
rowName() will often not be what you want to use any more... it will add quotes if the rowname contains dots

very shortly we will have the new rowPathElement(0) function which will return the true string